.evergreen/provision-atlas.sh

```diff
@@ -42,6 +42,6 @@ DATABASE=$DATABASE \
     $PYTHON_BINARY $SCAFFOLD_SCRIPT
 
 # If a search index configuration can be found, create the index
-if [ -f "$TARGET_DIR/indexConfig.json" ]; then
+if [ -d "$TARGET_DIR/indexConfig.json" ]; then
```

Comment:
Isn't `-d` directory and `-f` files?

Reply:
Oh yeah! I think this snuck in splitting out another PR.
```suggestion
if [ -f "$TARGET_DIR/indexConfig.json" ]; then
```